agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -112,6 +112,7 @@ function help {
     echo " -cs : CRI socket for Image Analyzer"
     echo " -cd : CRI-containerd socket for Image Analyzer"
     echo " -cv : custom volume for Image Analyzer"
+    echo " -b  : enable eBPF probe. 0 by default"
```

Comment:
Given that this flag doesn't take a parameter, you should remove the `0 by default` part.

Reply:
ah ok, makes sense thanks!